app/common/local.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+from flask import request
+
+
+def check_if_local() -> bool:
+    """
+    A helper function to check whether a request is being made from a local connection or on the
+    live website.
+    """
+    local = False
+
+    if os.environ.get("IS_LOCAL"):
+        local = (
+            request.remote_addr == "127.0.0.1"
+            or os.environ.get("VPN")
+            or request.host == "localhost"
```

Comment:
@y-himanen I need this like for pytest. Do I understand correctly that on prod it will be different? Just to make sure

Reply:
@rodriguesk This is clear. The question is `request.host == "localhost"` will be false on prod or not? I suppose it will be false, but I want just to double-check.

Reply:
yes, it should be.